src/gui/guiFormSpecMenu.cpp

```diff
@@ -66,8 +66,8 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 
 #define MY_CHECKGEOM(a,b)													\
 	if (v_geom.size() != 2) {												\
-		errorstream<< "Invalid pos for element " << a << "specified: \""	\
-			<< parts[b] << "\"" << std::endl;								\
+		errorstream<< "Invalid geometry for element " << a <<				\
+			"specified: \"" << parts[b] << "\"" << std::endl;				\
```

Comment:
Because of consistency with `MY_CHECKPOS`, `"specified: \""` should be moved back up.

Reply:
Moving `"specified: \""` back up makes it too long and it hits the backslash (using 4 space long tabs which the dev wiki code style guidelines suggest).  I could increase all the indents, but I don't know if I should.

Reply:
Oh, ok, then this seems ok as it currently is.